frontend/src/components/Login.tsx

```diff
@@ -0,0 +1,74 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { useState } from 'react';
+import { Link } from "react-router-dom";
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+    const navigate = useNavigate();
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        
+        axios.post( 'http://localhost:3001/login', {email, password})
```

Comment:
In the top level common directory, there is a package called schemas. Please use the appropriate zod schema to guarantee runtime type safety.